frontend/service.go

```diff
@@ -661,7 +661,7 @@ func (s *lwdStreamer) GetMempoolTx(exclude *walletrpc.Exclude, resp walletrpc.Co
 	}
 	for _, txid := range MempoolFilter(mempoolList, excludeHex) {
 		if tx, ok := (*mempoolMap)[txid]; ok {
-			if len(tx.Hash) > 0 {
+			if len(tx.Txid) > 0 {
```

Comment:
Aside: Under what circumstances could `mempoolMap` contain a `CompactTx` value at a key of `txid` (so the txid is known), but `tx.Txid` empty? That seems like a programming error.